WcaOnRails/app/models/competition.rb

```diff
@@ -1681,6 +1682,22 @@ def set_wcif!(wcif, current_user)
     end
   end
 
+  def set_wcif_competitor_limit!(wcif_competitor_limit, current_user)
+    unless current_user.can_admin_competitions? || !confirmed?
```

Comment:
Only thing I was worried about when I saw this PR. I appreciate the checks. I think it should be fine to change any field as long as it's the right time still.